git-cliff/src/lib.rs

```diff
@@ -300,6 +355,9 @@ fn process_repository<'a>(
 			previous_release.previous = None;
 			release.previous = Some(Box::new(previous_release));
 			previous_release = release.clone();
+			if recurse_submodules {
```

Comment:
Because previous release is needed, I moved this to the bottom of the for loop. Else it wouldn't be initialized.